### PR TITLE
Remove white spaces and bash label in code blocks

### PR DESCRIPTION
**Pull request by journey2w**
### Bugs Fixed ###
Remove white spaces and bash label in code blocks.

### CODE REVIEW

File: docs/open-source-project/reference/gbs/gbs-clone.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ Use the `gbs clone` subcommand to clone a Git repository into a new directory. U
...
+``````
 gbs clone [-h] [--upstream-branch <Upstream_Branch>]
           [--packaging-branch <Working_Branch>]
           [--all] [--depth <Depth>] <Repository> [<Directory>]
 ```
 
 - Mandatory parameter:
-  ```bash
+  ``````
   <Repository>        Specifies the path of target local repository or the URL
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/reference/gbs/gbs-clone.md]: Resolved by fixing a code block tag.

File: docs/open-source-project/reference/gbs/gbs.conf.md [hunk 2]
```diff
@@ -146,7 +165,8 @@ profile = profile.ivi
 
 When you specify the profile section with the `-P (--profile)` option in a GBS command, the specified profile configuration is applied:
 
-```bash
+```
+
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/reference/gbs/gbs.conf.md]: Done.

File: docs/open-source-project/reference/gbs/gbs-build.md [hunk 3]
```diff
@@ -92,53 +94,57 @@ To perform a basic build:
...
 
-  ```bash
+  ```
   $ gbs build -A i586 # build first and create build environment
   $ gbs build -A i586 --noinit # use --noinit to start building directly
   ```
 
-- Build with all uncommitted changes using the `--include-all` option.  
+- Build with all uncommitted changes using the `--include-all` option.
   In the following examples, the Git tree contains 1 modified file and 2 extra files:
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/reference/gbs/gbs-build.md]: Fixed.

File: docs/open-source-project/porting/kernel.md [hunk 4]
```diff
@@ -8,37 +8,37 @@ For information on how to set up the Tizen OS development environment, see [Sett
...
-1. Prepare the kernel source code for TM1 from `profile/mobile/platform/kernel/linux-3.10-sc7730`:  
+1. Prepare the kernel source code for TM1 from `profile/mobile/platform/kernel/linux-3.10-sc7730`:
    ```
    git: https://review.tizen.org/git/?p=profile/mobile/platform/kernel/linux-3.10-sc7730.git
    branch: accepted/tizen_mobile
    ```
 1. If your kernel source has been used to create binaries for another architecture, start by cleaning them up.
-1. Set up the `.config` file for TM1:  
-   ```bash
+1. Set up the `.config` file for TM1:
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/porting/kernel.md]: I think we don't need to insert an useless empty line. 
List will be displayed case by case.

----
When the system is build markdown files, the system provides a line break when there is a code block.
Also, the system apply the same style when there are items of unordered list.
Please see the final display of the following code:
CASE 1:
```
- item1
- time1
- item1
```
- item1
- time1
- item1

CASE 2:
```
- item2

- time2
- item2
```
- item2

- time2
- item2


CASE 3:
```
- item2

   paragraph 2
- time2
- item2
```
- item2

  paragraph 2
- time2
- item2

CASE 3: (same as CASE 2)
```
- item3

  paragraph 3

- time3
- item3
```
- item3

  paragraph 3

- time3
- item3

CASE 4: 
```
- item4 <br>
  paragraph 4
- time4
- item4
```

In this case, the comment is shown with browsing.  When you see the following code in .md file, you can't see an empty line between item4 and pragraph4.
- item4 <br>
  paragraph 4
- time4
- item4

File: docs/open-source-project/developing/building-all.md [hunk 5]
```diff
@@ -283,58 +283,58 @@ To clone the latest sources of all projects over HTTPS:
...
-   ```bash
+   ```
    $ sed -i 's/ssh://review.tizen.org/https://git.tizen.org//cgit/' .repo/manifests/_remote.xml
    ```
 
    This changes the 'fetch url' from `fetch="ssh://review.tizen.org/"` to `fetch="https://git.tizen.org/cgit/"`.
 
-4. Clone the specific snapshot source of all projects over HTTPS (optional).  
+4. Clone the specific snapshot source of all projects over HTTPS (optional).
    Replace the latest manifest with a snapshot manifest and make proper modifications by executing the following commands, as appropriate:
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/developing/building-all.md]: Fixed. I will keep two paragraphs.

File: docs/open-source-project/reference/mic/mic-faq.md [hunk 6]
```diff
@@ -1,37 +1,45 @@
...
 
 
-**Q**: How does MIC select packages? And how do I set the priority of a repository?  
-**A**: In general, MIC selects a higher version if 2 or more are available in all repositories. If the version is the same, a higher release number is preferred.  
-If you assign a priority to a repository, MIC prefers to select packages from a repository with a higher priority, even if a higher version is available in a repository with a lower priority. The default priority for a repository is 99, the range of repository priorities is 1~99, and the larger number has the lower priority.  
-To set the priority:  
+**Q**: How does MIC select packages? And how do I set the priority of a repository?
+
+**A**: In general, MIC selects a higher version if 2 or more are available in all repositories. If the version is the same, a higher release number is preferred.
+If you assign a priority to a repository, MIC prefers to select packages from a repository with a higher priority, even if a higher version is available in a repository with a lower priority. The default priority for a repository is 99, the range of repository priorities is 1~99, and the larger number has the lower priority.
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/reference/mic/mic-faq.md]: If you browse the changed file, you can see 1Q and 1A in a paragraph. 
If I changed the white spaces to an empty line, a new paragraph was formed. It is shown as ugly style.

If you think the current change is not matched with documentation rules, I will change the paragraph.

 I changed to use `<br>`.

File: docs/open-source-project/porting/multimedia.md [hunk 7]
```diff
@@ -37,7 +42,7 @@ The following list defines the major functions for the camera HAL interface:
 
   ```cpp
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/open-source-project/porting/multimedia.md]: `cpp` is okay. It depends on the build system that supports which kinds of languages.

File: docs/open-source-project/porting/multimedia.md [hunk 8]
```diff
@@ -738,10 +746,10 @@ Stream and device configuration:
   CONFIG_SND_SOC_I2C_AND_SPI = y
   CONFIG_SND_SOC_MC1N2=y
   ```
-- PulseAudio  
-  Version: 5.0  
+- PulseAudio
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/open-source-project/porting/multimedia.md]: This comment is not fixed, and these 2 lines are still merged into one. Is it intended?

**Original**
image

**In this PR**
image

File: docs/open-source-project/porting/multimedia.md [hunk 9]
```diff
@@ -738,10 +746,10 @@ Stream and device configuration:
...
+  Version: 5.0
   Website: [http://www.freedesktop.org/wiki/Software/PulseAudio](http://www.freedesktop.org/wiki/Software/PulseAudio)
-- ALSA  
+- ALSA
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/open-source-project/porting/multimedia.md]: This comment is not fixed, and these 2 lines are still merged into one. Is it intended?

**Original**
image


**In this PR**
image

File: docs/open-source-project/porting/multimedia.md [hunk 10]
```diff
@@ -777,7 +785,7 @@ There is no specific OAL for the multimedia player framework. The OAL interface
 
 ### References
 
-- Display driver configuration for the Samsung chipset  
+- Display driver configuration for the Samsung chipset
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/open-source-project/porting/multimedia.md]: This comment is not fixed, and these 2 lines are still merged into one. Is it intended?

**Comment by mijin85cho** [comment]
I checked the answers from you, and there are 3 questions which are not answered neither fixed for docs/open-source-project/porting/multimedia.md.
Would you please reply to that comments, too? Others are checked.

**Pull request merged**